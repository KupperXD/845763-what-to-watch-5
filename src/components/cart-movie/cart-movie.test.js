import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from "react-router-dom";

import CartMovie from "./cart-movie";

const noop = () => {};


it(`renders correctly`, () => {
  const cartMovie = renderer
    .create(
        <BrowserRouter>
          <CartMovie
            previewVideoLink={``}
            name={``}
            previewImage={``}
            id={1}
            isPlaying={true}
            onHoverCard={noop}
            onLeaveCard={noop}
          />
        </BrowserRouter>
    )
    .toJSON();

  expect(cartMovie).toMatchSnapshot();
});
