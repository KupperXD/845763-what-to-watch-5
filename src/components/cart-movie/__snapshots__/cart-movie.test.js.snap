// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseOut={[Function]}
  onMouseOver={[Function]}
>
  <a
    className="small-movie-card__link"
    href="/films/1"
    onClick={[Function]}
  >
    <div />
    <h3
      className="small-movie-card__title"
    >
      
    </h3>
  </a>
</article>
`;
